lib/matplotlib/axes/_axes.py

```diff
@@ -6476,22 +6476,22 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
             range of x.
 
         density : bool, default: False
-            If ``True``, the first element of the return tuple will
-            be the raw counts per bin normalized to form a probability density
-            so that the area under the histogram will integrate to 1, i.e.
-            ``np.sum(n * np.diff(bins)) == 1``.  (Computed by dividing the
-            raw counts ``n0`` by the total number of data points and
-            multiplying by the bin width: ``n = n0 / sum(n0) * np.diff(bins)``)
+            If ``True``, draw and return a probability density: each bin
+            will display the bin's raw count divided by the total number of
+            counts *and the bin width*
+            (``density = counts / (sum(counts) * np.diff(bins))``),
```

Comment:
This isn't right either, is it?  It should be `density = counts / sum(counts * np.diff(bins))`

Reply:
... no?

Reply:
sum(counts) is a scalar and diff(bins) a vector, aren't they?

Reply:
I guess both are the same in case of constant bin width. But in general it needs to be  
 `density = counts / (sum(counts) * np.diff(bins))`. So the way it's currently written is correct. (Maybe it helps thinking of `sum(counts)` as the total number of observations `N` and write it down as `1/N * counts / diff(bins)` ?)

Reply:
My apologies.  I was confusing myself.  This is correct.